### ISSUE TITLE
Make more levels

### ISSUE THREAD

**Issue by stevenhauser** [stevenhauser/i-have-to-return-some-videotapes#7]
Would be great to be able to progress throughout levels. Would also be great if the first level was a bit more thought out and less haphazard.

**Comment by andfaulkner** [comment]
Agreed, more levels please :). I'm really impressed by your game, you've done some amazing stuff with React and Redux.

**Comment by stevenhauser** [comment]
Thanks @andfaulkner! I'd like to make more levels and address some of these issues but I just haven't gotten around to it.
